### DISCUSSION TITLE
PCB hat idea

### DISCUSSION THREAD

**Discussion by Flourdeau** [probonopd/MiniDexed#378] [Ideas]
Hi everybody, today, i've made a raspberry pi hat (without the midi).
my idea is simple, making a hat to hookup the hardware with  jst plugs (2.54 pitched).
if anybody have any suggestions and advices, your're welcome.
Fred
![Photo View_2022-11-26](https://user-images.githubusercontent.com/56364029/204108891-fc5ca18a-0470-4a2f-9262-2af7572aafb5.svg)
![Photo View_2022-11-26 2](https://user-images.githubusercontent.com/56364029/204108896-593be5ef-f7fc-4bc8-a383-b7bf2ff6dad4.svg)
![PCB_PCB_mini dexed hat_2 1_2022-11-26 3](https://user-images.githubusercontent.com/56364029/204108905-5b4ba810-9a55-4653-b9e4-c8eedf3c3a9a.svg)
![Capture d’écran 2022-11-26 220919](https://user-images.githubusercontent.com/56364029/204109022-61f86ffe-958f-48b7-9901-4041ca3cb5ce.png)

[Schematic_mini dexed hat_2022-11-26.pdf](https://github.com/probonopd/MiniDexed/files/10097034/Schematic_mini.dexed.hat_2022-11-26.pdf)

**Comment by Flourdeau** [comment]
anyone know if placing a decoupling capacitors for the i2s and the i2c screen, is a good idea ?

greetz, fred

**Reply by BobanSpasic** [reply]
Decoupling capacitors are to be placed between Vcc+ and Gnd at the nearest position to ICs on a PCB (nowhere needed for this project).
The second use is against sparks from mechanical contacts (like encoders). Here you are missing one for the switch.

Do not put capacitors on signal lines.
